### PR TITLE
added header+4section+footer

### PR DESCRIPTION
**Pull request by sashaadazhii**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,19 +13,48 @@
...
+                        <li class="menu__item"><a href="" class="menu__link">about me</a></li>
+                        <li class="menu__item"><a href="" class="menu__link">servises</a></li>
+                        <li class="menu__item"><a href="" class="menu__link">my works</a></li>
+                        <li class="menu__item"><a href="" class="menu__link">clients</a></li>
+                    </ul>
+                </nav>
+
+                <div class="header-info">
+                    <div class="header-info__logo"></div>
+                    <div class="title title--header">free portfolio PSD template FOR GRAPHIC designerS</div>
```
Comment (Edifear) [inline index.html]: `h1...h6` for all titles?

File: index.html [hunk 2]
```diff
@@ -13,19 +13,48 @@
...
+                        quo, voluptates eum, cumque eos in ut quas modi assumenda dicta provident. Numquam ullam maiores
+                        eaque tenetur accusantium veritatis enim!</div>
+                    <div class="btn-wrapper">
+                        <div class="btn-container">
+                            <button class="btn">hire me</button>
+                        </div>
+                    </div>
+
+                    <div class="header-info__control">
+                        <object type="image/svg+xml" data="./assets/img/lnr-arrow-down.svg"></object>
```
Comment (Edifear) [inline index.html]: is this the same as ``?

File: style.scss [hunk 3]
```diff
@@ -20,15 +20,16 @@ $weight-bold: 700;
...
 $color1: #3677fd;
 $color2: #5a5a5a;
 $color-basic: #fff;
-$color-menu: #194277;
+$color-cookie: #194277;
 $color-bgr: #f1f8ff;
 
 $tablet: 768px;
-$mobile: 400px;
+$mobile: 500px;
```
Comment (Edifear) [inline style.scss]: make variables names more specific (e.g. `$mobile-screen-width`)

File: style.scss [hunk 4]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
-// .box { @include transform(rotate(30deg)); }
-//     @include font($family, $weight-regular, $fsize-title, $color1);
-// }
-
-
 
 .container {
     width: 1140px;
-    padding: 0 15px;
+    padding: 90px 15px;
```
Comment (Edifear) [inline style.scss]: vertical padding is not recommended for container, but works

File: style.scss [hunk 5]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+
+.title {
+    @include font($family, $fsize-title, $weight-light, $color1);
+    text-transform: capitalize;
+}
+
+.title--header {
+    @include font($family, $fsize-title, $weight-black, $color-basic);
+    text-transform: uppercase;
+     padding: 40px;
```
Comment (Edifear) [inline style.scss]: - line is not aligned
- try to make properties *order* more consistent (e.g. `position, display, margin, padding, font...`)

File: style.scss [hunk 6]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+    @include mobile {
+        font-size: $fsize-medium;
+    }
+}
+
+/*buttons*/
+.btn-wrapper {
+    padding: 50px 0;
+}
+    .btn-container {
```
Comment (Edifear) [inline style.scss]: - not aligned
- why do you need container for a button? (assume you could just make `display: inline-block;` for a button and style it properly

File: style.scss [hunk 7]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+            box-shadow: inset 0 0 2px 2px $color2;
+
+            .btn {
+                cursor: pointer;
+                color: $color2;
+            }
+        }
+    }
+
+    .btn-container--small {
```
Comment (Edifear) [inline style.scss]: use nesting `.btn-container { &--small { ... } }`

File: style.scss [hunk 8]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+            justify-content: center;
+
+            .cookies__desc {
+                @include font($family, $fsize-primary, $weight-regular, $color-basic);
+                padding: 0 50px;
+
+                @include mobile {
+                    padding: 0;
+                }
+            }
```
Comment (Edifear) [inline style.scss]: move it level above and use `&__desc { ...`

File: style.scss [hunk 9]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+                        text-align: center;
+                        width: 100%;
+                        height: 50vh;
+                        border-radius: 9px;
+
+                        .menu__item {
+                            color: $color2;
+                            padding: 15px;
+                        }
+                    }
```
Comment (Edifear) [inline style.scss]: not aligned

File: style.scss [hunk 10]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+                }
+    
+                &:hover + .menu,
+                &:active + .menu {
+                        display: flex;
+                        flex-direction: column;
+                        background-color: $color-basic;
+                        text-align: center;
+                        width: 100%;
+                        height: 50vh;
```
Comment (Edifear) [inline style.scss]: why `vh`?

File: style.scss [hunk 11]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+
+                        &:hover,
+                        &:active {
+                            color: $color1;
+                        }
+                    }
+                 }
+             }
+        }
+    }
```
Comment (Edifear) [inline style.scss]: not aligned

File: style.scss [hunk 12]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+
+        .header-info {
+            display: flex;
+            flex-flow: column nowrap;
+            align-items: center;
+            width: 820px;
```
Comment (Edifear) [inline style.scss]: fixed width?

File: style.scss [hunk 13]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+            max-width: 100%;
+            margin: 0 auto;
+
+            &__logo {
+                width:175px;
+                height: 175px;
+                border-radius: 50%;
+                background-color: $color-basic;
+                box-shadow: 0px 0px 10px 10px rgba(255,255,255,0.5);
```
Comment (Edifear) [inline style.scss]: remove `px` value is `0`

File: style.scss [hunk 14]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+        .header-info {
+            display: flex;
+            flex-flow: column nowrap;
+            align-items: center;
+            width: 820px;
+            max-width: 100%;
+            margin: 0 auto;
+
+            &__logo {
+                width:175px;
```
Comment (Edifear) [inline style.scss]: space after `:`

File: style.scss [hunk 15]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+.desc {
+    background-color: $color-bgr;
+    
+    .list {
+        display: flex;
+        flex-flow: row wrap;
+
+        &__item {
+            display: flex;
+            flex: 1 0 33%;
```
Comment (Edifear) [inline style.scss]: use `100 / 3`?

File: style.scss [hunk 16]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+       
+        .content {
+            @include font($family, $fsize-title, $weight-light, rgba(255,255,255,0.59));
+        
+            @include mobile {
+                font-size: $fsize-medium;
+            }
+        }
+        
+        .content--large {
```
Comment (Edifear) [inline style.scss]: use `.content { &--large { ... } }`

File: style.scss [hunk 17]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+            font-weight: $weight-bold;
+            color: $color-basic;
+        }
+        
+        @include mobile {
+            flex-direction: column;;
```
Comment (Edifear) [inline style.scss]: two `;;`

File: style.scss (hunk 13 above)
Comment (Edifear) [inline style.scss]: space after each comma `255, `

File: style.scss (hunk 14 above)
Comment (Edifear) [inline style.scss]: space after `: `

File: style.scss [hunk 18]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+                @include font($family, $fsize-primary, $weight-regular, $color-basic);
+                padding: 0 50px;
+
+                @include mobile {
+                    padding: 0;
+                }
+            }
+
+            @include mobile {
+                flex-direction: column;;
```
Comment (Edifear) [inline style.scss]: `;;`

File: style.scss [hunk 19]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+}
+    .btn-container {
+        min-width: 200px;
+        padding: 20px;
+        text-align: center;
+        background-color: $color-basic;
+        border-bottom-left-radius: 27px;
+        border-top-left-radius: 27px;
+        border-top-right-radius: 27px;
+        border-bottom-right-radius: 27px;
```
Comment (Edifear) [inline style.scss]: `border-radius` shortcut?

File: style.scss [hunk 20]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
+.container--small {
+    padding: 20px 15px;
+}
+
+.title {
+    @include font($family, $fsize-title, $weight-light, $color1);
+    text-transform: capitalize;
+}
+
+.title--header {
```
Comment (Edifear) [inline style.scss]:
```
.title {
  ...
  
  &--header { ... }
}

File: style.scss [hunk 21]
```diff
@@ -56,19 +57,317 @@ $mobile: 400px;
...
-    padding: 0 15px;
+    padding: 90px 15px;
     max-width: 100%;
     margin: 0 auto;
     text-align: center;
-}
\ No newline at end of file
+}
+
+.container--small {
```
Comment (Edifear) [inline style.scss]:
```
.container {
  ...
  
  &--header { ... }
}
```

File: index.html (hunk 2 above)
Comment (sashaadazhii) [reply index.html]: not sure that using  is correct here.
https://developer.mozilla.org/ru/docs/Web/SVG/Tutorial/Getting_Started

File: style.scss (hunk 4 above)
Comment (sashaadazhii) [reply style.scss]: how can I make it better with my html? Every section has this '90px' on top and bottom.

File: style.scss (hunk 5 above)
Comment (sashaadazhii) [reply style.scss]: shame on me! :)

File: style.css [hunk 22]
```diff
@@ -17,12 +17,250 @@ ul {
...
 /*end of variables*/
 /*------------------mixins------------------*/
-/*end of mixins*/
 .container {
   width: 1140px;
-  padding: 0 15px;
   max-width: 100%;
+  padding: 90px 15px;
   margin: 0 auto;
+<<<<<<< HEAD
```
Comment (Edifear) [inline style.css]: @sashaadazhii seems like you didn't compile css before pushing ;)

File: style.scss (hunk 10 above)
Comment (sashaadazhii) [reply style.scss]: because there is no mobile psd, so thi is on me)))

**Pull request merged**